packages/@okta/vuepress-site/docs/reference/api/roles/index.md

```diff
@@ -1277,6 +1277,808 @@ curl -v -X DELETE \
+
+##### Request parameters
+
+
+| Parameter        | Description                                            | Param Type  | DataType                                    | Required  |
+| :--------------- | :----------------------------------------------------- | :---------- | :------------------------------------------ | :-------- |
+| roleType         | `type` of a Role                                       | URL         | [Role Type](#role-types)                    | TRUE      |
+| notificationType | `type` of a notification                               | URL         | [Notification Type](#notification-type)    | TRUE      |
+
+##### Response parameters
+
+
+[Subscription](#subscription-object)
+
+##### Request example
+
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/roles/SUPER_ADMIN/subscriptions/IWA_AGENT"
+```
+
+##### Response example
+
+
+```json
+{
+    "notificationType": "IWA_AGENT",
+    "channels": [
+        "email"
+    ],
+    "status": "subscribed",
+    "_links": {
+        "unsubscribe": {
+            "href": "http://rain.okta1.com:1802/api/v1/roles/SUPER_ADMIN/subscriptions/IWA_AGENT/unsubscribe",
+            "hints": {
+                "allow": [
+                    "POST"
+                ]
+            }
+        },
+        "self": {
+            "href": "http://rain.okta1.com:1802/api/v1/roles/SUPER_ADMIN/subscriptions/IWA_AGENT"
+        }
+    }
+}
+```
+
+#### Subscribe to a specific type
+
+<ApiLifecycle access="ea" />
+
+<ApiOperation method="post" url="/api/v1/roles/${roleType}/subscriptions/${notificationType}/subscribe" />
+
+Make a Role subscribes to a specific type
+
+##### Request parameters
+
+
+| Parameter        | Description                                            | Param Type  | DataType                                 | Required  |
+| :--------------- | :----------------------------------------------------- | :---------- | :--------------------------------------- | :-------- |
+| roleType         | `type` of a Role                                       | URL         | [Role Type](#role-types)                 | TRUE      |
+| notificationType | `type` of a notification                               | URL         | [Notification Type](#notification-type) | TRUE      |
+
+##### Response parameters
+
+
+``` http
+HTTP/1.1 200 OK
+```
+
+##### Request example
+
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/roles/SUPER_ADMIN/subscriptions/IWA_AGENT/subscribe"
+```
+
+##### Response example
+
+
+``` http
+HTTP/1.1 200 OK
+```
+
+#### Unsubscribe to a specific type
+
+<ApiLifecycle access="ea" />
+
+<ApiOperation method="post" url="/api/v1/roles/${roleType}/subscriptions/${notificationType}/unsubscribe" />
```

Comment:
something which I think we clarify in our product help docs is that every time a role-wide subscription changes, it results in overriding individual subscriptions. It may be worth adding as a note here as well: https://help.okta.com/en/prod/Content/Topics/Settings/set_default_notifications.htm